pages/search.vue

```diff
@@ -48,14 +48,21 @@ export default {
           keywords: keywords && R.split(',', keywords),
           tags: tags && R.split(',', tags),
         }
-      }, 500)
+      }, 100)
```

Comment:
Can we do 300? 100 is a bit too flashy.
```suggestion
      }, 300)
```